### PR TITLE
Added Tailwind CSS to Two row Carousel Section

### PR DESCRIPTION
**Pull request by tanujbordikar**
Solves Issue no #132

### CODE REVIEW

File: sections/dev__carousel--two-rows.liquid [hunk 1]
```diff
@@ -1,54 +1,63 @@
...
         {{- section.settings.cta_text -}}
       </a>
     </div>
   </div>
 
   <!-- Upper Row Carousel -->
-  <div class="carousel__row carousel__row--upper">
+  {%- assign image_url_width = section.settings.image_url_width -%}
+  {%- assign image_url_height = section.settings.image_url_height -%}
+  <div class="carousel__row {{ section.settings.carousel__row }} carousel__row--upper {{ block.settings.carousel__row--upper }}">
```
[the remark this answers is not in the document]
Comment (mihir-bombay-studio) [reply sections/dev__carousel--two-rows.liquid]: image
I mean the upper row styles and lower row styles text boxes are used outside the block scope

[the remark this answers is not in the document]
Comment (tanujbordikar) [reply sections/dev__carousel--two-rows.liquid]: Okay @mihir-bombay-studio Sir understood, I have moved the styles for both row within the block scope can you review it now!

**Comment by mihir-bombay-studio** [comment]
@tanujbordikar No for now let's keep the block level styling inside the block itself

**Comment by tanujbordikar** [comment]
> @tanujbordikar No for now let's keep the block level styling inside the block itself

Okay Sir.

**Review by mihir-bombay-studio — APPROVED** [review]

**Pull request merged**